guides/source/configuring.md

```diff
@@ -711,6 +711,53 @@ Sets fallback behavior for missing translations. Here are 3 usage examples for t
 
 Is a boolean value which controls whether the `full_message` error format can be overridden at the attribute or model level in the locale files. This is `false` by default.
 
+ActiveRecord and ActiveModel can both use i18n to format their full messages. By default, full message is formatted as `"%{attribute} %{message}"` where
+
+* `%{attribute}` is the name of the attribute being validated.
+* `%{message}` the message defined for the validation rule. It can be defined in the class or in i18n locales.
+
+This example overrides the format of full messages of every attribute on an Active Model and defines a specific full message format for the `:age` attribute. Same rules apply for Active Records.
```

Comment:
I think we should integrate these paragraphs.  What do you think of the following?

```suggestion
Controls whether the [`Error#full_message`][ActiveModel::Error#full_message] format can be overridden in an i18n locale file. Defaults to `false`.

When set to `true`, `full_message` will look for a format at the attribute and model level of the locale files. The default format is `"%{attribute} %{message}"`, where `attribute` is the name of the attribute, and `message` is the validation-specific message. The following example overrides the format for all `Person` attributes, as well as the format for a specific `Person` attribute (`age`).
```

We would also need to add a URL reference right below the example:

```
[ActiveModel::Error#full_message]: https://api.rubyonrails.org/classes/ActiveModel/Error.html#method-i-full_message
```